frontend/src/store/index.js

```diff
@@ -258,6 +258,22 @@ function mapAccessRestrictionForDisplay ({ definition, accessRestriction: { valu
   }
 }
 
+// Return first item with classification supported, if no item has classification supported
+// return first item with classifiction undefined, if no item matches these requirements,
+// return first item in list
+function firstItemMatchingVersionClassification (items) {
+  let defaultItem = find(items, { classification: 'supported' })
+  if (!defaultItem) {
+    defaultItem = find(items, machineImage => {
+      return machineImage.classification === undefined
+    })
+  }
+  if (!defaultItem) {
+    defaultItem = head(items)
+  }
+  return defaultItem
```

Comment:
just a suggestion (early exit)

```suggestion
  let defaultItem = find(items, { classification: 'supported' })
  if (defaultItem) {
    return defaultItem
  }
  
  defaultItem = find(items, machineImage => {
    return machineImage.classification === undefined
  })
  if (defaultItem) {
    return defaultItem
  }
  
  defaultItem = head(items)
```